tests/testthat/test_format_external_packages.R

```diff
@@ -0,0 +1,51 @@
+context("External package support Import/Export")
+require("datasets")
+
+
+extensions <- c("bib", "bmp", "gexf", "gnumeric", "jpeg", "jpg", "npy", "png", "sss", "sdmx", "tiff")
+read_fn <- function(ext) {
+    switch(ext,
+          bib = "bib2df::bib2df",
+          bmp = "bmp::read.bmp",
+          gexf = "rgexf::read.gexf",
+          gnumeric = "gnumeric::read.gnumeric.sheet",
+          jpeg = "jpeg::readJPEG",
+          jpg = "jpeg::readJPEG",
+          npy = "RcppCNPy::npyLoad",
+          png = "png::readPNG",
+          sdmx = "sdmx::readSDMX",
+          sss = "sss::read.sss",
+          tiff = "tiff::readTIFF")
+}
+
+test_that("External read functions without an import method", {
+    message <- "%s format not supported. Consider using the '%s()' function"
+    for (ext in extensions) {
+        file <- file.path(tempdir(), paste0("test.", ext))
+        file.create(file)
+        on.exit(unlink(file))
+        expect_error(import(file), sprintf(message, ext, read_fn(ext)), fixed=TRUE)
```

Comment:
Messages are locale-specific, so it would be better to just have this be an `expect_error()` that doesn't match against the specific error message.

Reply:
Done.